### DISCUSSION TITLE
When is the official M1 support going to be released?

### DISCUSSION THREAD

**Discussion by alexhanh** [Homebrew/discussions#649] [Everyday usage] [answered]
I've tried to search for information everywhere. The issues https://github.com/Homebrew/brew/issues/10152 and https://github.com/Homebrew/brew/issues/7857 cover some of the development with a conclusion that the release is being worked on, but without further info. Have I missed something?

Would it be possible to get a status update along with some estimated timeline?

Thank you!

**Comment by carlocab** [comment, marked as answer]
This should answer your question re a status update: https://github.com/Homebrew/discussions/discussions/624

As for timelines, there isn't really one. A lot of it depends on packaged software supporting M1. That's not really in Homebrew's control, nor can Homebrew predict when that's going to happen.

**Comment by SMillerDev** [comment]
Eventually, we're all volunteers and it's really hard to plan people's free time and motivation.
